### PR TITLE
Micronaut: Code completion support for values of 'Mapping' annotation.

### PR DESCRIPTION
**Pull request by dbalek**
Adding code completion support for the `from` and `to` values of  the `io.micronaut.context.annotation.Mapper.Mapping` annotation.

### CODE REVIEW

**Review by MartinBalin — APPROVED** [review]

File: enterprise/micronaut/src/org/netbeans/modules/micronaut/completion/MicronautJavaMappingPropertiesCompletion.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import org.netbeans.modules.micronaut.db.Utils;
+
+/**
+ *
+ * @author Dusan Balek
+ */
+public class MicronautJavaMappingPropertiesCompletion implements Processor {
+
+    private static final Set<String> supportedAnnotationTypes = Set.of("io.micronaut.context.annotation.Mapper.Mapping");
+    private Reference<ProcessingEnvironment> processingEnv;
```
Comment (sdedic) [inline enterprise/micronaut/src/org/netbeans/modules/micronaut/completion/MicronautJavaMappingPropertiesCompletion.java:50]: This ref seems to be unused.

**Review by sdedic — APPROVED** [review]

File: enterprise/micronaut/src/org/netbeans/modules/micronaut/completion/MicronautJavaMappingPropertiesCompletion.java (hunk 1 above)
Comment (dbalek) [reply enterprise/micronaut/src/org/netbeans/modules/micronaut/completion/MicronautJavaMappingPropertiesCompletion.java]: Good catch. Thanks.

File: java/java.editor/src/org/netbeans/modules/editor/java/JavaCompletionCollector.java [hunk 2]
```diff
@@ -651,7 +652,7 @@ public Completion createAttributeValueItem(CompilationInfo info, String value, S
                 }
             }
             Builder builder = CompletionCollector.newBuilder(label)
-                    .kind(Completion.Kind.Text)
+                    .kind(Completion.Kind.Value)
                     .sortText(value)
```
Comment (mbien) [inline java/java.editor/src/org/netbeans/modules/editor/java/JavaCompletionCollector.java:656]: something in this PR, possibly this snippet is causing a regression: https://github.com/apache/netbeans/issues/7522

Comment (mbien) [reply java/java.editor/src/org/netbeans/modules/editor/java/JavaCompletionCollector.java:656]: sorry for the noise, I can't reproduce it with the nightly build. This means my dev build didn't clean properly while switching branches or something similar - my mistake.

**Pull request merged**